### PR TITLE
12.0 invoice error checking and updating

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

**Review by fredrikarvas-vertel — APPROVED** [review]
This looks good to me.

**Review by fredrikarvas-vertel — CHANGES_REQUESTED** [review]
Please add a Jira and change the header to invoice. I do not think it has to do with interpreter.

File: outplacement_invoice/__manifest__.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
         'data/cron.xml',
         'data/test.xml',
         'views/account_invoice_view.xml',
+        'data/server_action.xml'
```
Comment (draal-af) [inline outplacement_invoice/__manifest__.py:42]: Are we missing the data file?

**Review by draal-af — CHANGES_REQUESTED** [review]

**Review by S-Arnshed-Capgemini — APPROVED** [review]

File: outplacement_invoice/data/server_action.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="update_outplacement_invoices" model="ir.actions.server">
+        <field name="name">Update all outplacement invoices</field>
+        <field name="model_id" ref="base.model_ir_actions_server"/>
+        <field name="state">code</field>
+        <field name="groups_id" eval="[(4, ref('base.group_system'))]"/>
+        <field name="code">
+for order in env['sale.order'].search([]):
```
Comment (nils-vertel) [inline outplacement_invoice/data/server_action.xml:9]: Commented code, please write a line on why we need to keep it

**Pull request merged**